package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "glue",
   "description": "Server composer for hapi.js",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Comment:
Many times for prs its best not to do the version, and the owner will change it appropriately when they are ready to publish it (tho I can't speak for @csrl ).